modules/op-release-notes-1-20-0.adoc

```diff
@@ -0,0 +1,225 @@
+// This module is included in the following assemblies:
+// * release_notes/op-release-notes-1-20.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="op-release-notes-1-20-0_{context}"]
+= Release notes for {pipelines-title} 1.20
+
+With this update, {pipelines-title} General Availability (GA) 1.20 is available on {OCP} 4.15 and later versions.
+
+[id="new-features-1-20-0_{context}"]
+== New features
+
+In addition to fixes and stability improvements, the following sections highlight what is new in {pipelines-title} 1.20:
+
+* {pipelines-title} is designed for environments running in Federal Information Processing Standards (FIPS) mode. When deployed on {OCP} configured for FIPS mode, the platform uses the {op-system-base-full} cryptographic libraries that have been submitted to National Institute of Standards and Technology (NIST) for FIPS validation. For more information about enabling OpenShift Container Platform FIPS support, see the link:https://docs.redhat.com/en/documentation/openshift_container_platform/4.18/html/installation_overview/installing-fips[OpenShift Container Platform documentation].
+
+* {pipelines-shortname} adds anti-affinity rules to High Availability (HA) setups, ensuring that replicas are not actively deployed on the same node. This improves system resilience and availability by distributing replicas across multiple nodes, reducing the risk of a single node failure affecting multiple replicas. This improves service availability and resilience, better load balancing, and overall system performance.
+
+* * With this update, you can independently control whether the {pipelines-shortname} Operator creates Role-Based Access Control (RBAC) resources and Trusted Certificate Authority (CA) bundle ConfigMaps in your cluster. Two new options in the `TektonConfig` custom resource (CR) allow you to configure this behavior:  
++
+** `createRbacResource`: Enables or disables creation of RBAC resources.  
```

Comment:
for new features, we need to specify under particular component similar to 1.19. For example - this should be under operator section